content/updates/2021-05/4-Product-completeness-in-the-API.md

```diff
@@ -7,11 +7,12 @@ pim_announcement_audience:
 # Retrieve product completeness through API
 ::: meta-data type="Improvement" features="Productivity" available="late May" in="EE,GE" link-to-doc="https://api.akeneo.com/concepts/products.html#product"
 
-When connecting your PIM to other applications through the API, you can now ask for the product completeness whenever retrieving products. Thanks to this piece of information, you can easily push forward the most enriched products on your different channels.
+ You can now retrieve your products' completeness in the REST API product data returns. This turns in handy when you need to connect your PIM to third-party systems. For instance, you can then use it to select the most enriched products before you share them to your different channels.
```

Comment:
@NolwennP did we get it right?

Reply:
Product data doesn't "return in the REST API". @camillebourjade quick call to clarify this?

Reply:
```suggestion
You can now get your products’ completeness whenever retrieving products through the REST API. It gives an even more thorough vision of the products on your third-party applications. For instance, it can be used to rank or highlight the most complete products on the different sales channels.
```